[Pulsar Functions] User class constructor throws exception

**Describe the bug**

A few days ago, I could use this command to create a function instance correctly, 
```
./bin/pulsar-admin functions create --retain-ordering --jar /Downloads/pulsar-util-function-api-pack-0.10.2-all.jar --classname ru.alfabank.api.utils.JournalBatchParseFunction --output-serde-classname ru.alfabank.api.utils.JournalBatchSerde --inputs test-input-topic --output test-output-topic --name test-serde-name
```

but now I re-use the same command, the creation fails, the error is as follows:

```
12:00:50.493 [public/default/test-serde-xiaolong-0] INFO  org.apache.pulsar.functions.instance.JavaInstanceRunnable - Load JAR: /tmp/pulsar_functions/public/default/test-serde-xiaolong/0/pulsar-util-function-api-pack-0.10.2-all.jar
12:00:50.548 [public/default/test-serde-xiaolong-0] WARN  org.apache.pulsar.common.nar.NarClassLoader - /var/folders/qz/g2fh76dx5rj4q_fbkfh1q3th0000gn/T/pulsar-nar/pulsar-util-function-api-pack-0.10.2-all.jar-unpacked does not contain META-INF/bundled-dependencies!
12:00:50.549 [public/default/test-serde-xiaolong-0] INFO  org.apache.pulsar.functions.instance.JavaInstanceRunnable - Initialize function class loader for function test-serde-xiaolong at function cache manager
12:00:50.553 [public/default/test-serde-xiaolong-0] ERROR org.apache.pulsar.functions.instance.JavaInstanceRunnable - [public/default/test-serde-xiaolong:0] Uncaught exception in Java Instance
java.lang.RuntimeException: User class constructor throws exception
	at org.apache.pulsar.functions.utils.Reflections.createInstance(Reflections.java:131) ~[pulsar-functions-utils.jar:2.5.0-SNAPSHOT]
	at org.apache.pulsar.functions.instance.JavaInstanceRunnable.setupJavaInstance(JavaInstanceRunnable.java:195) ~[pulsar-functions-instance.jar:?]
	at org.apache.pulsar.functions.instance.JavaInstanceRunnable.run(JavaInstanceRunnable.java:242) [pulsar-functions-instance.jar:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_201]
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_201]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_201]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_201]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_201]
	at org.apache.pulsar.functions.utils.Reflections.createInstance(Reflections.java:123) ~[pulsar-functions-utils.jar:2.5.0-SNAPSHOT]
	... 3 more
Caused by: java.lang.NoClassDefFoundError: com/ibm/as400/access/AS400Bin4
	at ru.alfabank.utils.eq.journal.JournalBatchParser.<init>(JournalBatchParser.java:24) ~[?:?]
	at ru.alfabank.utils.eq.journal.JournalBatchParser.defaultParser(JournalBatchParser.java:50) ~[?:?]
	at ru.alfabank.api.utils.JournalBatchParseFunction.<init>(JournalBatchParseFunction.java:13) ~[?:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_201]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_201]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_201]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_201]
	at org.apache.pulsar.functions.utils.Reflections.createInstance(Reflections.java:123) ~[pulsar-functions-utils.jar:2.5.0-SNAPSHOT]
	... 3 more
```

**To Reproduce**

This problem does not seem to be stable and reproducible.

**Expected behavior**

Can always create functions correctly.

**Screenshots**

![image](https://user-images.githubusercontent.com/20965307/67563111-a3438180-f752-11e9-8f23-e7fe1eecac38.png)


**Desktop (please complete the following information):**
 - OS: Mac OS

**Additional context**
Add any other context about the problem here.

--- a/conf/functions_worker.yml
+++ b/conf/functions_worker.yml
@@ -38,7 +38,7 @@ zooKeeperOperationTimeoutSeconds: 30
 ################################
 
 numFunctionPackageReplicas: 1
-downloadDirectory: /tmp/pulsar_functions
+downloadDirectory: download/pulsar_functions
 # Classname of Pluggable JVM GC metrics logger that can log GC specific metrics
 # jvmGCMetricsLoggerClassName: 
 

